docs/modules/migrate/pages/dmt-run.adoc

```diff
@@ -0,0 +1,53 @@
+
+Make sure you have everything you need before you start, as detailed in the xref:migrate:dmt-run.adoc#before-you-begin[] section below.
+
+== Before You Begin
+
+If you haven't already done so, disconnect all client applications from the source cluster.
+
+Hazelcast also recommends that the source cluster is put in a `PASSIVE` state before you start the migration, as described in the  xref:maintain-cluster:cluster-member-states#changing-a-clusters-state.adoc[Changing a Cluster's State] topic. If you do not do this, the DMT might not migrate data that changed during the migration.
+
+Ensure that you have installed the following:
+
+* https://docs.docker.com/get-docker/[Docker^]
+* https://docs.hazelcast.com/clc/latest/install-clc[Hazelcast Command-Line Client (CLC)^]
+
+Before running the migration, you need the following information:
+
+* Your Operating System
+* Your processor architecture
+* The binary that is suitable for your machine
++
+You can find DMT binaries in the _bin_ folder of the extracted DMT package. The binaries are in the format `dmt_[platform]_[arch]`.  Use the ``arm64`` binary for ARM, and the `amd64` binary for Intel.
+
+== Start the Migration
+
+To run the migration, complete the following steps:
+
+. Open a terminal
+. Navigate to the folder containing the extracted DMT package
+. Enter the following command:
++
+[source,shell]
+----
+./bin/dmt_[platform]_[arch] --config migration.yaml start migration_config --yes --log.path migration.log
+----
+
+[NOTE]
+====
+. `--log.path migration.log` specifies that the migration logs are saved to the _migration.log_ file on completion of the migration. For further information on viewing the migration details, see the xref:migrate:data-migration-tool.adoc#view-result[View Migration Results]
+. The DMT will attempt to connect to the migration cluster indefinitely. This means that it can appear to hang if unable to connect. To avoid this, you can set a timeout for the connection attempt using the `--timeout` flag.  For further information on the `--timeout` flag, refer to the https://docs.hazelcast.com/clc/5.3.5/configuration#clc-configuration-with-command-line-parameters[CLC Configuration with Command-line Parameters^] section of the Hazelcast CLC documentation. 
+. On MacOS, you might need to allow the `dmt*` binary to run. If the command is rejected, go to the *Privacy & Security* settings on your device and update them to allow the binary. After updating the settings, retry the command, and select *Open* when prompted
+====
+
+== Track the Migration
+
+You can use the DMT `status` command to track the migration. 
+
+For further information on the available DMT commands, see the xref:migrate:dmt-command-reference.adoc[DMT Command Reference].
```

Comment:
I think it's not in this PR but we need to update the command reference. --config flag for dmt migration cluster config to connect to migration cluster is missing, also --yes flag etc.(not sure if we should add it but good to do )

Reply:
I didn't do the Command Reference last time, so it didn't occur to me!

@srknzl do you want me to pick this up, or would you prefer that the team continues to maintain this page?

Reply:
IMO it would be the best if the user created a named configuration using `clc config add` instead of providing the configuration path directly. In that case they don't need to provide the configuration with each command. CLC will display a configuration selector if necessary.

Reply:
> IMO it would be the best if the user created a named configuration using clc config add instead of providing the configuration path directly. In that case they don't need to provide the configuration with each command. CLC will display a configuration selector if necessary.

yeah that'll be very good. What if they don't have clc though? In any case, I think we should have --config flag in the command reference because it's available for use and I don't see a point in hiding it. 

@rebekah-lawrence @yuce and I can do that, thank you

Reply:
@rebekah-lawrence note: one possible update I see is that adding a note about what @yuce said, basically they can create a named clc config so that they don't have to specify "migration.json" config path all the time via --config flag. 

@yuce is it possible to call "dmt config add"?

Reply:
I have added a note, called it `dmt config add` for now along with a x-ref to the DMT Command Reference assuming the `dmt config` command will be added there. Can easily change if this isn't implemented as dmt-specific.

Reply:
@srknzl 
DMT has the same `config` commands with CLC.

Reply:
Rebekah gave a link to dmt command reference. We should update DMT command reference and state that clc commands like "clc config" are also usable with dmt

Reply:
https://github.com/hazelcast/hz-docs/pull/981